test/swivel/Sig.sol

```diff
@@ -37,7 +37,7 @@ library Sig {
   /// @return v The verification bit
   /// @return r First 32 bytes
   /// @return s Next 32 bytes
-  function split(bytes memory sig) internal pure returns (uint8, bytes32, bytes32) {
+  function split(bytes memory sig) private pure returns (uint8, bytes32, bytes32) {
```

Comment:
we have many `internal` methods. if you are worried about derived contracts then why not change them all to `private`. seems half-baked to just change 1?